README.md

```diff
@@ -28,7 +28,7 @@ This will:
 * spin up grafana/prometheus for monitoring, see `http://localhost:3000`.
 * spin up a bootstrap nwaku node.
```

Comment:
maybe we are missing the command to build the python script?

running just
`docker-compose up -d`

gives

```
Service rest-traffic has neither an image nor a build context specified. At least one must be provided.
```


how can we build the image? any chance it can be done automatically with `docker-compose up -d`?

Reply:
That's weird! It worked for me initially, that's why I added
https://github.com/waku-org/waku-simulator/blob/dbf8b7a7bfe6d46560bd7b13f1bb986cbc555f41/docker-compose.yml#L64-L66

I'll check again, but yes. The idea is for it to be built automatically